css/styles.css

```diff
@@ -99,3 +95,81 @@ nav.navbar {
     color: #f7f7f7;
     font-size: 16px;
 }
+
+/* OUR STORY section */
+
+#our_story, #review {
+    min-height: 600px; 
+}
+
+#our_story .row {
+    margin-top: 30px;
+}
+
+#our_story h1 {
+    font-family: 'Lobster', sans-serif;
+    color: #f8512f;
+    font-size: 6em;
+    text-shadow: 1px 1px 5px #0a0a0a;
+}
+
+#our_story h3 {
+    font-family: 'Raleway', sans-serif;
+    font-weight: 700;
+    font-size: 2em;
+    margin-top: 0;
+}
+
+#our_story p {
+    font-family: 'Raleway', sans-serif;
+    font-weight: 300;
+}
+
+.story_year {
+    font-family: 'Playfair Display', sans-serif;
+    font-size: 200px;
+    font-weight: 700;
+    z-index: -1;
+    position: absolute; 
+    color: rgba(149, 147, 147, 0.2);
+}
+
+@media (max-width: 350px) {
+    #our_story {
+        justify-content: center;
+    }
```

Comment:
this is no issue right now, but after PR has been merged. move this media query to the bottom. looks great though.